package.json

```diff
@@ -31,7 +31,6 @@
     "@mapbox/geojson-merge": "^1.1.1",
     "@mapbox/point-geometry": "^0.1.0",
     "@mapbox/togeojson": "^0.16.2",
-    "@mapbox/vector-tile": "^1.3.0",
```

Comment:
Should ` lib/ThirdParty/mapbox__vector-tile` also be deleted if this package is removed?